packages/sku/src/services/vite/helpers/createConfig.ts

```diff
@@ -57,6 +61,7 @@ export const createViteConfig = ({
       ssr: configType === 'ssr' || configType === 'ssg',
       manifest: configType === 'client',
       ssrManifest: false,
+      assetsDir: skuContext.publicPath.replace(/(^\/|\/$)/g, ''),
```

Comment:
The leading slash is important here as it distinguishes between a root-relative path and a document-relative path.

```suggestion
      assetsDir: skuContext.publicPath.replace(/\/$/, ''),
```

Reply:
I added the forward slash removal because Vite doesn't support leading slashes in the assetDir `patterns can be neither absolute nor relative paths. If you want your files to be stored in a subdirectory, write its name without a leading slash like this: subdirectory/pattern.`.